src/bin/build-keycloak-theme/build-keycloak-theme.ts

```diff
@@ -14,11 +15,15 @@ type ParsedPackageJson = {
 
 const reactProjectDirPath = process.cwd();
 
-const doUseExternalAssets = process.argv[2]?.toLowerCase() === "--external-assets";
+const doUseExternalAssets = argv("external-assets");
 
 const parsedPackageJson: ParsedPackageJson = require(pathJoin(reactProjectDirPath, "package.json"));
 
-export const keycloakThemeBuildingDirPath = pathJoin(reactProjectDirPath, "build_keycloak");
+const buildPath = argv("buildPath") ? String(argv("buildPath")) : "build";
```

Comment:
the string "buildPath" shouldn't be repeated (for maintainability).  
Variables should be scoped as strictly as possible so that the code is easier 
to reason about.  

```ts
import {  assert } from "tsafe/assert";

//...

const buildPath= (()=>{

    const argName = "buildPath";

    const value = argv(argName);

    assert(value !== true, `expected --${argName} <path>, no <path> was provided`);

    return value || "build";

})();
```